js/ui/projectManager.js

```diff
@@ -505,12 +515,66 @@ define([
       if (!keepRetained) chrome.storage.local.remove("retainedProject");
       this.render();
     },
-    
+
     getPaths: function() {
       return Object.keys(this.pathMap);
     }
   };
 
+  // Resize sidebar
+  // min / max sidebar width
+  var MIN_WIDTH = 100;
+  var MAX_WIDTH = 500;
+
+  function setupSidebarResize(element, document) {
+    // Sidebar resizing
+    var isResizing = false;
+    // make sure width is not set in element `style`
+    element.style.width = null;
+
+    var startSidebarResize = function (evt) {
+      if (evt.target.hasClass('project-resizer')) {
```

Comment:
You keep referencing this element, but I'm not aware of having one. Did you forget to commit the index.html changes that added it?